cmd/loadTests.go

```diff
@@ -969,7 +969,7 @@ func userJourneyThread(frameworkMap *sync.Map, threadWaitGroup *sync.WaitGroup,
 			app, err := framework.AsKubeDeveloper.HasController.CreateApplicationWithTimeout(ApplicationName, usernamespace, 60*time.Minute)
 			applicationCreationTime := time.Since(startTimeForApplication)
 			if err != nil {
-				logError(3, fmt.Sprintf("Unable to create the Application %s: %v", ApplicationName, err))
+				logError(31, fmt.Sprintf("Unable to create the Application %s: %v", ApplicationName, err))
```

Comment:
Could you add a comment somewhere explaining these code changes? If it is some refactor, not directly connected with this time skew effort, could you make a different PR for it?

Reply:
This error numberring is not directly related to this time skew effort, so I will revert to numberring the logError with consequitive numbers and we can spare the next pr for the diffrent error numberring